README.md

```diff
@@ -372,6 +372,15 @@ cp sample-dcrdata.conf ~/.dcrdata/dcrdata.conf
 Then edit dcrdata.conf with your dcrd RPC settings. See the output of `dcrdata
 --help` for a list of all options and their default values.
 
+### Edit postgresql settings in drcdata.conf file. 
```

Comment:
I'm still uncertain about including this (there are a lot of dcrdata.conf settings that should be reviewd before running), but not describing the `psql` commands (`CREATE USER...`, `CREATE DATABASE...`, etc.) that were the initial hang up.

Reply:
got it